lib/galaxy/webapps/galaxy/api/tool_shed_repositories.py

```diff
@@ -202,6 +231,47 @@ def index( self, trans, **kwd ):
         return tool_shed_repository_dicts
 
     @expose_api
+    def install( self, trans, payload, **kwd ):
+        """
+        POST /api/tool_shed_repositories/install
+        Initiate the installation of one or more repositories.
+
+        :param encoded_kwd (required): The encoded dict describing the repositories to be installed.
```

Comment:
encoded_kwd isn't used in this function ... the docs don't really line up with the code below.